README.md

```diff
@@ -42,7 +42,7 @@ If you have an existing extension and want to update it to use the latest standa
 you can run the following in the extension's directory:
 
 ```console
-$ solidus extension .
+$ bundle exec solidus extension .
```

Comment:
Thanks for clarifying this!

I think we may want to also clarify in the section above on generating a new extension, that this gem needs to be installed globally (i.e. `gem install solidus_dev_support` in order for the the `solidus` executable to be added to the path. The installation instructions only focus on adding this as a dependency of an existing extension, which I think is causing some of the confusion.